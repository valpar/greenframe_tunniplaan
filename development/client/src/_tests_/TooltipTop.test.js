import React from 'react';
import { render, fireEvent, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import TooltipTop from '../components/UI/Tooltip/TooltipLarge';

describe('TooltipTop', () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<TooltipTop />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('displays the provided message', () => {
    const message = "Test Tooltip Message";
    render(<TooltipTop message={message} />);
    expect(screen.getByText(message)).toBeInTheDocument();
  });
});