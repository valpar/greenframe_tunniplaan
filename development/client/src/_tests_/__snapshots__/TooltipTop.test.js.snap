// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TooltipTop matches snapshot 1`] = `
<div
  className="relative w-auto h-auto border border-red-500/20 rounded bg-white error-tp-shadow"
>
  <div
    className="absolute w-4 h-4 rotate-45 bg-white error-tp-a-shadow -bottom-2 left-1/2 -translate-x-1/2"
  />
  <div
    className="absolute w-6 h-4 bg-white bottom-0 left-1/2 -translate-x-1/2"
  />
  <div
    className="py-3 px-4 bg-white font-normal text-base text-center md:text-lg"
  >
    <p />
  </div>
</div>
`;
